cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -446,7 +446,7 @@ def validate_job_id(self):
                 if not self.public:
                     debug_message += " for your user account email"
                 logstash_message(self.app, {'origin': 'dispatcher-call-back', 'event': 'unauthorized-user'})
-                raise RequestNotAuthorized("user not authorized to download the requested product", debug_message=debug_message)
```

Comment:
oh, sorry I overlooked and noticed while testing something else, this is not just called when download, right?

Reply:
Exactly! Also when `run_analysis', and perhaps in the future it will be during `call_back` (and for emails I saw something...?)

Reply:
it is already used in all kinds of places  where `not authorized to download the requested` is not right, since user is not trying to download anything. Maybe previous, more generic, message was better in this case.

Reply:
The current one is actually quite generic (`Request not authorized`) unless I misunderstood

Reply:
it says there in the first argument `not authorized to download the requested`, right?

Reply:
That is the old message, I have now re-written

[https://github.com/oda-hub/dispatcher-app/blob/03b473cdc785d5387f6e4e5f772b7dfd3cd8de07/cdci_data_analysis/flask_app/dispatcher_query.py#L449](https://github.com/oda-hub/dispatcher-app/blob/03b473cdc785d5387f6e4e5f772b7dfd3cd8de07/cdci_data_analysis/flask_app/dispatcher_query.py#L449)

Reply:
ok, I looked in a wrong place